swizzle/main.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Perhaps solved via docs, but I wonder if there's a better name for this playbook? It's unclear that this playbook is tied to the `provision.py` script.

Reply:
I don't think this is exclusively tied to `provision.py`, as I think we can pass "wardroom_action" values to ansible-playbook via `-e` parameter when running the playbooks. This allows us to use the same set of playbooks for install, upgrade, adding nodes, etc.

Reply:
Ah! That's a good point.

Reply:
All of the playbooks may be called exclusively, this is simply a common entrypoint that someone _may_ use...not mandatory.  Yes, docs are needed here.